specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2023-07-01-preview/ResourceHealth.json

```diff
@@ -0,0 +1,2167 @@
+                  "description": "Recommended action locale for the service health event."
+                }
+              }
+            },
+            "faqs": {
+              "type": "array",
+              "x-ms-identifiers": [],
+              "description": "Frequently asked questions for the service health event.",
+              "items": {
+                "$ref": "#/definitions/faq"
+              }
+            },
+            "isHIR": {
+              "type": "boolean",
+              "description": "It provides information if the event is High incident rate event or not."
+            },
+            "enableMicrosoftSupport": {
+              "type": "boolean",
+              "description": "Tells if we want to enable or disable Microsoft Support for this event."
+            },
+            "description": {
+              "type": "string",
+              "description": "Contains the communication message for the event, that could include summary, root cause and other details."
+            },
+            "platformInitiated": {
+              "type": "boolean",
+              "description": "Is true if the event is platform initiated."
+            },
+            "enableChatWithUs": {
+              "type": "boolean",
+              "description": "Tells if we want to enable or disable Microsoft Support for this event."
+            },
+            "priority": {
+              "type": "integer",
+              "format": "int32",
+              "description": "Priority level of the event. Has value from 0 to 23. 0 is the highest priority. Service issue events have higher priority followed by planned maintenance and health advisory. Critical events have higher priority followed by error, warning and informational. Furthermore, active events have higher priority than resolved."
+            },
+            "lastUpdateTime": {
+              "type": "string",
+              "description": "It provides the Timestamp for when the health impacting event was last updated.",
+              "format": "date-time"
+            },
+            "hirStage": {
+              "type": "string",
+              "description": "Stage for HIR Document"
+            },
+            "additionalInformation": {
+              "type": "object",
+              "description": "Additional information",
+              "properties": {
+                "message": {
+                  "type": "string",
+                  "description": "Additional information Message"
+                }
+              }
+            },
+            "duration": {
+              "type": "integer",
+              "description": "duration in seconds",
+              "format": "int32"
+            },
+            "impactType": {
+              "type": "string",
+              "description": "The type of the impact"
+            },
+            "maintenanceId": {
+              "type": "string",
+              "description": "Unique Id for Planned maintenance event"
```

Comment:
Doc style - they should be "planned maintenance events" (all lowercase) - no need to uppercase this in English docs.

Reply:
I will fix this, thank you!